ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/validation/ModelConformanceValidatorSpec.scala

```diff
@@ -145,9 +343,70 @@ class ModelConformanceValidatorSpec extends AnyWordSpec with Matchers with Mocki
 }
 
 object ModelConformanceValidatorSpec {
-  private val aKeyMaintainer = "maintainer"
+  private val inputContractId = "#inputContractId"
+  private val inputContractIdStateKey = makeContractIdStateKey(inputContractId)
+  private val contractId1 = "#someContractId"
+  private val contractIdStateKey1 = makeContractIdStateKey(contractId1)
+  private val inputContractKey = "inputContractKey"
+  private val inputContractKeyMaintainer = "inputContractKeyMaintainer"
   private val aKey = "key"
+  private val aKeyMaintainer = "maintainer"
   private val aDummyValue = TransactionBuilder.record("field" -> "value")
+  private val aTemplateId = "dummyPackage:DummyModule:DummyTemplate"
+
+  private val aSubmissionSeed = ByteString.copyFromUtf8("a" * 32)
+  private val ledgerEffectiveTime =
+    ZonedDateTime.of(2021, 1, 1, 12, 0, 0, 0, ZoneOffset.UTC).toInstant
+
+  private val aContractIdStateValue = {
```

Comment:
Is it possible to make it more concise?

Reply:
I'm afraid not. That's the verbosity of protobuf :/